src/summary.js

```diff
@@ -0,0 +1,115 @@
+            totalBuilds: 0,
+            totalBuildsApproved: 0,
+            totalBuildsUnreviewed:0,
+            totalBuildsFailed:0,
+            totalBuildsRequestingChanges:0,
+            totalSnapshots: 0,
+            totalSnapshotsRequestingChanges: 0,
+            totalSnapshotsUnreviewed: 0,
+            totalSnapshotsReviewed: 0,
+            totalComparisons: 0,
+        },_summary)
+        if (cursor) {
+            urlParams['page[cursor]'] = cursor
+        }
+        let _builds = await axios.get('/builds', { params: urlParams, responseType: 'json' }).then((res) => {
+            if (res.status == 200) {
+                return JSON.parse(res.data)
+            } else {
+                throw res.data
+            }
+        })
+        for (let build of _builds.data) {
+            let createdAt = new Date(build.attributes["created-at"])
+            if (isAfter(createdAt, endDate) || isBefore(createdAt, startDate)) {
+                done = true;
+                break
+            }
+            if (isBefore(createdAt, endDate) && isAfter(createdAt, startDate)) {
+                summary['totalBuilds']++;
+                if (build['attributes']['review-state'] == 'approved') {
+                    summary['totalBuildsApproved']++
+                }
+                if(build['attributes']['review-state'] == 'unreviewed'){
+                    summary['totalBuildsUnreviewed'] ++
+                }
+                if(build['attributes']['review-state'] == null){
+                    summary['totalBuildsFailed'] ++;
+                }
+                if(build['attributes']['review-state'] == 'changes_requested'){
+                    summary['totalBuildsRequestingChanges'] ++;
+                }
+                summary['totalSnapshots'] += build['attributes']['total-snapshots']
+                summary['totalSnapshotsRequestingChanges'] += build['attributes']['total-snapshots-requesting-changes']
+                summary['totalSnapshotsUnreviewed'] += build['attributes']['total-snapshots-unreviewed']
+                summary['totalSnapshotsReviewed'] += (build['attributes']['total-snapshots'] - build['attributes']['total-snapshots-unreviewed'])
+                summary['totalComparisons'] += build['attributes']['total-comparisons']
+            }
+        }
+        if (done) {
+            return summary
+        } else {
+            if (_builds.data.length > 0) {
+                return buildSummary(_builds.data[_builds.data.length - 1].id,summary)
+            } else {
+                return summary;
+            }
+        }
+    }
+
+    let summary = await buildSummary()
+    summary = {
+        ...summary,
+        browsers,
+        projectName,
+        startDate,
+        endDate
+    }
+    if(!fs.existsSync('Summary') ){
+        fs.mkdirSync('Summary',{recursive:true})
+    }
+    fs.writeFileSync(`Summary/${summary.projectName}-${Date.now()}.json`,JSON.stringify(summary,undefined,2))
```

Comment:
Cosmetic: Can we add "week" or "prefix" to the report so its easier to distinguish. Or maybe group it in a folder.